### ISSUE TITLE
Gem doesn't allow Rails 6.1.6 to be used

### ISSUE THREAD

**Issue by jfahrenkrug** [ibmdb/ruby-ibmdb#135]
The gemspec states "rails <= 6.1". That means that 6.1.1 or 6.1.6 are not allowed. I'm getting this error when trying to use Rails 6.1.6 with ibm_db 5.3.1:

```Bundler could not find compatible versions for gem "activerecord":
  In Gemfile:
    ibm_db (= 5.3.1) was resolved to 5.3.1, which depends on
      activerecord (>= 5.0.7, <= 6.1)

    rails (~> 6.1.7) was resolved to 6.1.7, which depends on
      activerecord (= 6.1.7)
```

Could we instead specify `< 6.2`? I think that would allow all 6.1.x versions up to (but not including) 6.2.

**Comment by praveen-db2** [comment]
@jfahrenkrug sure will fix this and let you know

**Comment by jfahrenkrug** [comment]
@praveen-db2 Thank you very kindly :)

**Comment by jfahrenkrug** [comment]
@praveen-db2 I can confirm this is fixed, thanks! It works with Rails 6.1.7.
